### PR TITLE
update automatic_updates...

### PR DESCRIPTION
**Pull request by szaimen**
to disable notifications for new app or software updates

### CODE REVIEW

**Comment by szaimen** [comment]
I've already tested the command but not together with occ_command...
(I've only tried it with occ_command_no_check but it should work nonetheless)

**Comment by szaimen** [comment]
What this does is it disables the notification about every single app update for admins and also disables notifications about Nextcloud-Updates so that only our notifiactions come through ;)
Also these are not needed because we run the update script every week and update NC and all apps

File: static/automatic_updates.sh [hunk 1]
```diff
@@ -32,6 +32,7 @@ In the next step you will be able to choose to proceed or exit."
 
 if [[ "yes" == $(ask_yes_or_no "Do you want to enable automatic updates?") ]]
 then
+    occ_command config:app:set updatenotification notify_groups --value="[]"
     touch $VMLOGS/update.log
```
Comment (enoch85) [inline static/automatic_updates.sh]: Please run the permissions script here to make sure the update.log permissions are correct.

Comment (szaimen) [reply static/automatic_updates.sh]: do you mean this one?
https://github.com/nextcloud/vm/blob/master/static/setup_secure_permissions_nextcloud.sh
I don't get completely why but okay...

Comment (enoch85) [reply static/automatic_updates.sh]: Nvm, I was thinking wrong. :)

**Comment by enoch85** [comment]
What if you would like to disable automatic updates? Then you might want to enable the regular notifications again.

**Comment by szaimen** [comment]
you can easily do that by just selecting a group here:
image

**Comment by szaimen** [comment]
We are only deleting the default admin group here.

**Comment by enoch85** [comment]
The `occ command` works, but I don't know the consequences except what you're saying.

Do you think this is ready for merge?

**Comment by enoch85** [comment]
OK, just tested, and it works.

**Pull request merged**